### PR TITLE
Code Cleanup

### PR DESCRIPTION
**Pull request by Jahrme**
- Cleanup the `Makefile` and fix a path.
- Remove an unused variable from `RawBitmapReader.c`.
- Cleanup `main.c` so that structure is more consistent and variables are clearer.

### CODE REVIEW

File: BitmapEngine/src/main.c [hunk 1]
```diff
@@ -3,75 +3,81 @@
...
+        char results_name[BUFF_SIZE];
+        /* where the results are being stored */
+        snprintf(results_name, BUFF_SIZE, "%s_RESULTS.csv", bitmap_file);
+
+        double time;
+
+        /* run compression here */
+        if (strcmp(format, "UNSTRIPED") == 0)
+        {
+            time = compress(bitmap_file, UNSTRIPED, BBC, n);
```
Comment (smburdick) [inline BitmapEngine/src/main.c]: When I compile this I get an error saying `n` is undeclared. Please test your code before submitting a review.

File: BitmapEngine/src/raw-bitmap-reader/RawBitmapReader.c [hunk 2]
```diff
@@ -477,7 +477,6 @@ void compressColumn(int col, int threadNum) {
 
     unsigned int min = 0;
     min--;
-    int optimal=0;
 
     for(i=0; i<runs; i++) {
```
Comment (smburdick) [inline BitmapEngine/src/raw-bitmap-reader/RawBitmapReader.c]: add space between for variables/operators in loop (see style guide for example)

File: BitmapEngine/src/raw-bitmap-reader/RawBitmapReader.c [hunk 3]
```diff
@@ -521,7 +520,6 @@ void compressColumn(int col, int threadNum) {
 
         if(format==VAL && NUM_SEGS==-1) {
```
Comment (smburdick) [inline BitmapEngine/src/raw-bitmap-reader/RawBitmapReader.c:521]: same spacing issue. please fix this everywhere.

File: BitmapEngine/src/main.c [hunk 4]
```diff
@@ -3,75 +3,81 @@
...
 int main(int argc, char *argv[])
 {
-
     setbuf(stdout, NULL);
-    // TODO: this function is overloaded. I think rather than using flags to tell the function what to do,
-    // move each function to a separate one to simplify the logic of this code.
-    // e.g., bformat, bcompress, bquery
 
-    if (argc > 2 && (strcmp(argv[1], "F") == 0 || strcmp(argv[1], "C") == 0 || strcmp(argv[1], "Q") == 0))
+    /* FORMAT */
```
Comment (smburdick) [inline BitmapEngine/src/main.c]: after all, I decided I prefer this style

File: BitmapEngine/src/main.c [hunk 5]
```diff
@@ -3,75 +3,81 @@
...
+    if (argc == 3 && strcmp(argv[1], "F") == 0)
```
Comment (smburdick) [inline BitmapEngine/src/main.c]: move `{` up

Comment (smburdick) [reply BitmapEngine/src/main.c]: fix everywhere

File: BitmapEngine/src/main.c [hunk 6]
```diff
@@ -3,75 +3,81 @@
...
     {
+        char* bitmap_file = argv[2];
```
Comment (smburdick) [inline BitmapEngine/src/main.c]: `char *bitmap_file`

File: BitmapEngine/src/main.c [hunk 7]
```diff
@@ -3,75 +3,81 @@
...
+        char* format = argv[3];
+        int striped = strcmp(format, "STRIPED");
+        int unstriped = strcmp(format, "UNSTRIPED");
+
+        if (striped != 0 && unstriped != 0) return -1;
+
+        char* bitmap_file = argv[2];
 
+        char results_name[BUFF_SIZE];
+        /* where the results are being stored */
```
Comment (smburdick) [inline BitmapEngine/src/main.c]: pointless comment

**Review by smburdick — CHANGES_REQUESTED** [review]

File: BitmapEngine/src/raw-bitmap-reader/RawBitmapReader.c (hunk 2 above)
Comment (Jahrme) [inline BitmapEngine/src/raw-bitmap-reader/RawBitmapReader.c:481]: Would you like me to style-check the entire `RawBitmapReader.c` file?
All I did in there was remove the `optimal` variable that was not being used; I had not planned on doing a style-audit but I can.

Comment (smburdick) [reply BitmapEngine/src/raw-bitmap-reader/RawBitmapReader.c:481]: not necessary. Disregard this comment.

**Comment by Jahrme** [comment]
With the possible exception of performing a style-audit on `RawBitmapReader.c` (see comment above), all of your comments should now be addressed.

I was confused by your report of errors as I did not remember getting any errors; I eventually realized that I had stashed some of my last changes resulting in them not getting pushed (n.b., if you `git stash`, make sure to `git stash pop` before submitting the PR); I have since pushed them at 674b0d7 and confirmed that things are working as expected by pulling this branch onto `db-capstone2` where it ran without issue.

File: BitmapEngine/src/main.c (hunk 1 above)
Comment (Jahrme) [reply BitmapEngine/src/main.c]: Fixed at 674b0d7.

**Pull request merged**